packages/front-end/components/Radix/HelperText.tsx

```diff
@@ -63,24 +63,22 @@ export function RadixStatusIcon({
   }
 }
 
-export default function HelperText({
-  children,
-  status,
-  size = "md",
-  ...otherProps
-}: {
-  children: string | string[];
-  status: Status;
-  size?: "sm" | "md";
-} & MarginProps) {
+export default forwardRef<
+  HTMLDivElement,
+  {
+    children: string | string[];
+    status: Status;
+    size?: "sm" | "md";
+  } & MarginProps
+>(function HelperText({ children, status, size = "md", ...otherProps }, ref) {
   return (
     <Text color={getRadixColor(status)} size={getRadixSize(size)}>
-      <Flex gap="1" {...otherProps}>
+      <Flex gap="1" {...otherProps} ref={ref}>
```

Comment:
Less feedback and more of a question - it's not initially clear to me how it's decided which element gets the ref passed to it and the props spread into it - do you have any insight on this?

Reply:
I think it usually is the one that 'makes more sense'. So if we are doing a Input that has a text for a helper and other components, we would expect the ref to be the input. But I also see an argument for it being the whole container in the scenarios where what we want to do is related to calculating its size/positioning.

So yeah, I am also not super sure. Right now I decided to mainly follow where the overall props are being spread, or adding the ref to the internal Radix component.